gateway/cli/src/main.rs

```diff
@@ -31,10 +31,12 @@ pub enum Commands {
     /// Ganerate gateway configuration
     /// NOTE: This command can only be used on a local gateway
     GenerateConfig {
+        /// Address to which the LN rpc webserver will bind
+        lnrpc_bind_address: SocketAddr,
         /// Address to which the API webserver will bind
-        bind_address: SocketAddr,
+        webserver_bind_address: SocketAddr,
         /// URL under which the API will be reachable
-        announce_address: Url,
+        api_announce_address: Url,
```

Comment:
Shouldn't these two both begin with `api_` or `webserver_` since they refer to the same thing?

Reply:
going with `api_` to standardize both config fields